### ISSUE TITLE
Add accessible link colours for dark backgrounds

### ISSUE THREAD

**Issue by lyubomir-popov** [canonical/vanilla-framework#4632]
I'm revisiting the problem of accessible link colours for dark backgrounds again. To pass  WCAG 2.0 rules about contrast against surrounding text, we are forced to use a colour that is difficult to read. Unless we decide it ok to fail the WCAG reqs to improve readability, I would suggest sticking to that  for now. Here are the values, can we please make them available in the framework?:

![image](https://user-images.githubusercontent.com/2741678/214831249-5452eb25-3905-4caa-8504-6613270e8d75.png)

For reference, Here's a related working group [discussion ](https://github.com/canonical/vanilla-framework/issues/3350) prpoising a change to the inverted link documentation, and the addition of an `.is-dark` class.

We will discuss this in the Accessibility guild and hopefully reach a conclusion whether we can afford to breach the outdated (but still in force) [WCAG requirements](https://www.w3.org/WAI/WCAG21/Techniques/general/G183), which is something most other sites do - including github and google chrome's settings pages for example.

For what it's worth, following thenew APCA recommendations is also not ideal, because links become almost indistinguishable from white text, thereby failing the https://www.w3.org/WAI/WCAG21/Techniques/general/G183 technique.  
![image](https://user-images.githubusercontent.com/2741678/214832326-8eec1d23-d02f-4b1e-902f-fd9e7f90b0d1.png)

**Comment by bartaz** [comment]
Duplicate of #4633
